app/assets/javascripts/discourse/tests/integration/components/reviewable-refresh/flag-reason-test.gjs

```diff
@@ -0,0 +1,137 @@
+import { render } from "@ember/test-helpers";
+import { module, test } from "qunit";
+import ReviewableFlagReason from "discourse/components/reviewable-refresh/flag-reason";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+
+module(
+  "Integration | Component | reviewable-refresh | flag-reason",
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    test("renders with basic arguments", async function (assert) {
+      const score = { type: "spam", title: "This is spam", count: 3 };
+
+      await render(
+        <template><ReviewableFlagReason @score={{score}} /></template>
+      );
+
+      assert
+        .dom(".review-item__flag-reason")
+        .exists("renders the flag reason container");
+      assert
+        .dom(".review-item__flag-reason")
+        .hasClass("--spam", "applies correct CSS class for spam type");
+      assert
+        .dom(".review-item__flag-reason")
+        .containsText("This is spam", "displays the title");
+      assert
+        .dom(".review-item__flag-count")
+        .exists("renders the flag count element");
+      assert
+        .dom(".review-item__flag-count")
+        .containsText("3", "displays the count value");
+    });
+
+    test("renders count when count is greater than 0", async function (assert) {
```

Comment:
Is this example needed? Count field is already checked in the previous one

Reply:
Ooops, forgot to delete that one when I was cleaning up. 🙂